### PR TITLE
Adding context mapping: REF

### PR DESCRIPTION
**Pull request by Bargenish**
## Status
- [ ] In Progress
- [X] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/22527

## Description
Adding context mapping

## Minimum version of Demisto
- [ ] 4.5.0
- [ ] 5.0.0
- [ ] 5.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [X] No

## Must have
- [X] Tests
- [ ] Documentation 

## Demisto Partner?
- [ ] The title must be in the following format: **[YOUR_PARTNER_ID] short description**

### CODE REVIEW

**Comment by yaakovi** [comment]
@kirbles19 added a new context key, please review description and changelog

File: Integrations/integration-qualys.yml [hunk 1]
```diff
@@ -1110,6 +1111,8 @@ script:
       description: Scan ID
     - contextPath: Qualys.Scan.Reference
       description: Sacn ref
+    - contextPath: Qualys.Scan.REF
+        description: Scan REF
```
Comment (yaakovi) [inline Integrations/integration-qualys.yml]:
```suggestion
      description: Scan REF
```

File: Integrations/integration-qualys.yml [hunk 2]
```diff
@@ -1110,6 +1111,8 @@ script:
       description: Scan ID
     - contextPath: Qualys.Scan.Reference
       description: Sacn ref
```
Comment (yaakovi) [inline Integrations/integration-qualys.yml]: there's a typo here (all over the integration.
please update the description to something more informative (currently both paths has the same description)

**Comment by kirbles19** [comment]
@Bargenish Done.

**Pull request merged**